Kernel/Output/PDF/Ticket.pm

```diff
@@ -173,11 +173,8 @@ sub GeneratePDF {
     # Get customer info.
     my $CustomerUserObject = $Kernel::OM->Get('Kernel::System::CustomerUser');
     my %CustomerData;
-    if ( $Interface{Customer} ) {
-        %CustomerData = $CustomerUserObject->CustomerUserDataGet(
-            User => $Param{UserID},
-        );
-    }
+    my %PrintingCustomerData;
+
     elsif ( $Ticket{CustomerUserID} ) {
```

Comment:
Shouldn't this be an 'if' instead of an 'elsif' ?

Reply:
Yes, it should - strange, I thought I pushed the correction.